net2/logger.js

```diff
@@ -163,8 +171,10 @@ module.exports = function (component) {
     }
   }
 
+  // winston 1.1.2 use different ordering on log levels
```

Comment:
we use 2.4.0 in .node_modules
https://github.com/firewalla/fnm.node8.aarch64/blob/master/winston/package.json

default order:
https://github.com/firewalla/fnm.node8.aarch64/blob/master/winston/lib/winston/config/npm-config.js